features/step_definitions/book-index-page.rb

```diff
@@ -1,40 +1,43 @@
 
-Given(/^there are (\d+) books in the database$/) do |arg1|
-    expect(books).to eq 100
-end
+Given(/^there are (\d+) books in the database$/) do |amount|
+  amount.to_i.times { create(:book) }
+end 
 
 When(/^I visit the root url$/) do
-  pending # express the regexp above with the code you wish you had
+  visit('/') 
```

Comment:
You'll be using this puppy a lot. I like how it's structured in a vague way. In your cucumber tests, you'll often need to test from the root url.